src/store/modules/job/actions.ts

```diff
@@ -765,6 +765,125 @@ const actions: ActionTree<JobState, RootState> = {
+            'systemJobEnumId': job.systemJobEnumId,
+            'tempExprId': job.frequency, // Need to remove this as we are passing frequency in SERVICE_TEMP_EXPR, currently kept it for backward compatibility
+            'maxRecurrenceCount': '-1',
+            'parentJobId': job.parentJobId,
+            'runAsUser': 'system', //default system, but empty in run now.  TODO Need to remove this as we are using SERVICE_RUN_AS_SYSTEM, currently kept it for backward compatibility
+            'recurrenceTimeZone': this.state.user.current.userTimeZone
+          },
+          'statusId': "SERVICE_PENDING",
+          'systemJobEnumId': job.systemJobEnumId
+        } as any
+
+        
+        if (job?.runtimeData?.shopifyConfigId) {
+          const shopifyConfig = this.state.user.shopifyConfigs.find((config: any) => {
+            return config.shopId === shopId;
+          })
+
+          params['shopifyConfigId'] = shopifyConfig.shopifyConfigId;
+          params['jobFields']['shopId'] = shopId;
+        }
+
+        // checking if the runtimeData has productStoreId, and if present then adding it on root level
+        job?.runtimeData?.productStoreId?.length >= 0 && (params['productStoreId'] = payload.eComStoreId)
+        job?.priority && (params['SERVICE_PRIORITY'] = job.priority.toString())
+
+        // assigning '' (empty string) to all the runtimeData properties whose value is "null"
+        job.runtimeData && Object.keys(job.runtimeData).map((key: any) => {
+          if (job.runtimeData[key] === 'null') job.runtimeData[key] = ''
+        })
+        jobParams.push({ ...job.runtimeData, ...params, jobId: job.jobId });
+      })
+    })
+
+    Promise.allSettled(jobParams.map(async (param: any) => {
+      const resp: any = await JobService.scheduleJob(param);
+      if(resp.status === 200 && !hasError(resp)){
+        // Removing the scheduled job
+        dispatch('removeBulkJob', param.jobId);
+        return Promise.resolve(resp);
+      } else {
+        failedJobs++;
+        return Promise.reject(resp);
+      }
+    })).then((resps: any) => {
+      resps.some((resp: any) => {
+        if(resp.status === "rejected") {
+          return showToast(translate("Failed to schedule service(s)", {count: failedJobs}))
+        } else {
+          return showToast(translate('Services have been scheduled'))
+        }
+      })
+    })
+  },
+  setBulkJobData({ commit, state }, payload) {
+    let bulkJobs = JSON.parse(JSON.stringify(state.bulk.jobs));
+    const value = payload.value, type = payload.type;
+    if (payload.global) {
+      commit(types.JOB_BULK_DATA_UPDATED, { type, value });
+      bulkJobs = bulkJobs.map((job: any) => {
+        // handling special case for slow frequency jobs
+        if (type === 'frequency' && job.freqType === 'slow') {
+          showToast(translate("Some jobs have slow frequency type, hence, maximum frequency will be set automatically"))
+          return ({...job, [type]: 'EVERYDAY'})
```

Comment:
Here we have hardcoded the frequency for slow jobs, but what if we have added another maximum frequency for slow jobs?

Also, suppose we have globally selected `HOURLY` frequency that can be applied to slow jobs as well, but then also we will change it to everyday.

Reply:
Updated sir